cosmo_tester/test_suites/broker_management/conftest.py

```diff
@@ -0,0 +1,112 @@
+import os
+import time
+
+import pytest
+
+from cosmo_tester.framework.test_hosts import BootstrapBasedCloudifyManagers
+from cosmo_tester.framework import util
+
+
+@pytest.fixture()
+def brokers(cfy, ssh_key, module_tmpdir, attributes, logger):
+    for _brokers in _get_brokers(cfy, ssh_key, module_tmpdir,
+                                 attributes, logger):
+        yield _brokers
+
+
+@pytest.fixture()
+def broker(cfy, ssh_key, module_tmpdir, attributes, logger):
+    for _brokers in _get_brokers(cfy, ssh_key, module_tmpdir, attributes,
+                                 logger, instance_count=1):
+        yield _brokers[0]
+
+
+class JustHosts(BootstrapBasedCloudifyManagers):
```

Comment:
i'd make a `MediumLinuxImage` in test_hosts, put the two methods (the ones other than bootstrap) from `BootstrapBasedCloudifyManagers` into it, have `BootstrapBasedCloudifyManagers` inherit that, and use that new class here directly

then we don't have the weird effect of "bootstrap based adds some behaviour, but here we also get rid of some of it"

and a more descriptive name i guess :P